src/scripts/site.js

```diff
@@ -1033,7 +1034,7 @@ var Dashboard = {
             }, appRouter.showSelectServer = function() {
                 AppInfo.isNativeApp ? Dashboard.navigate("selectserver.html") : Dashboard.navigate("login.html")
             }, appRouter.showWelcome = function() {
-                Dashboard.navigate("login.html")
+                AppInfo.isNativeApp ? Dashboard.navigate("selectserver.html") : Dashboard.navigate("login.html")
```

Comment:
If you want a ternary why so big?
```suggestion
                Dashboard.navigate(AppInfo.isNativeApp ? "selectserver.html" : "login.html")
```

Reply:
same here, pulled from older working version